WBooks/MainTabBar/LibraryTab.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LibraryTab.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 24/05/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class LibraryTab: UITabBarController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTabBar()
+        addNavBarButtons()
+    }
+    
+    private func addNavBarButtons() {
+        self.title = "NAVIGATION_BAR_TITLE".localized()
+        self.setNavigationBarTitle("NAVIGATION_BAR_TITLE".localized(), font: UIFont.systemFont(ofSize: 20, weight: UIFont.Weight.medium), color: UIColor.white)
+        //Left notification button
+        navigationItem.leftBarButtonItem = UIBarButtonItem.notificationButton()
+        //Right search button
+        navigationItem.rightBarButtonItem = UIBarButtonItem.searchButton()
+    }
+    
+    private func configureTabBar() {
+        tabBar.barTintColor = .white
+        let library = MainMenuController()
+        library.tabBarItem = UITabBarItem()
+        library.tabBarItem.title = "Library"
```

Comment:
This strings should be on the localized file that you've created in the other card.